### PR TITLE
Mobile version milestone 03

### PR DESCRIPTION
**Pull request by jenadiusnicholaus**
Mobile version milestone 03:
In this milestone:

1. Added contact me feature
2. Styled the the form to add validation
3. Added Tablets version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -294,9 +294,42 @@ <h1> Skils</h1>
...
         </section>
         <!-- end of about -->
-        <!-- <section  class="get-in-touch">
-            <h1>Get in</h1>
-        </section> -->
+        <section  class="get-in-touch">
+            <h3 hidden>heading</h3>
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="/" method ="post">
```
Comment (Deepakdanger) [inline index.html]: - [ ] Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).
- Action value should be similar to `https://formspree.io/f/{form_id}`.Creade `form_ID` on formspree.io. Let's fix this.


method value equals POST

File: index.html [hunk 2]
```diff
@@ -294,9 +294,42 @@ <h1> Skils</h1>
...
+                <div class="input-control">
+                    <input class="fullname-control" type="text" id="full-name" placeholder="Full name" required>
```
Comment (Deepakdanger) [inline index.html]: - [ ] Name input (can be first name and last name field depending on the template used) has `maxlength="30"` attribute. Let's add `maxlength` here.

File: index.html [hunk 3]
```diff
@@ -294,9 +294,42 @@ <h1> Skils</h1>
...
+                </div>
+                <div class="input-control">
+                    <input class="emaol-control" type="email" id="email" placeholder="Email address" required>
+                </div>
+                <div class="input-control">
+                    <textarea name="content" id="contents"   rows="4" placeholder="Enter text here" required></textarea>
```
Comment (Deepakdanger) [inline index.html]: - [ ] Message input has `maxlength="500"` attribute. Let's add `maxlength` here.

File: index.html [hunk 4]
```diff
@@ -294,9 +294,42 @@ <h1> Skils</h1>
...
+                </div>
+                <div class="get-in-touch-button">
+                    <div hidden></div>
+                <button  class="btn-get-in-touch" type="submit">Get in touch</button>
+                    <div hidden></div>
+                </div>
+            </form>
+        </section>
+
+        <section class="footer">
```
Comment (Deepakdanger) [inline index.html]: - [ ] Instead of using the `section` tag here, the `footer` tag is best to use as a semantic tag. Kindly use the `footer` tag here.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @jenadiusnicholaus, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jenadiusnicholaus) [reply index.html]: Is it recommenced to do this?

File: index.html (hunk 2 above)
Comment (jenadiusnicholaus) [reply index.html]: I am following a figma design  that i chosen not the picture on the requirements, as the design differ from one other, for my design no  first name and last name fields

File: index.html (hunk 4 above)
Comment (jenadiusnicholaus) [reply index.html]: This is OK, i will fix that.

File: index.html (hunk 3 above)
Comment (jenadiusnicholaus) [reply index.html]: I will fix that too

**Comment by jenadiusnicholaus** [comment]
Thanks for you comments

File: index.html (hunk 1 above)
Comment (thecodechaser) [inline index.html]: - [ ] As requested by the previous reviewer, Please add action value to your form so it should be submitted to [formspree service](https://formspree.io/html) when users click on the submit button. Action value should be similar to `https://formspree.io/f/{form_id}.` Create `form_ID` on formspree.io.

File: index.html (hunk 2 above)
Comment (thecodechaser) [inline index.html]: - [ ] As requested by the previous reviewer, Please add a `maxLength` attribute to your name field and set value to 30, So your name input field does not accept values more than 30 characters.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @jenadiusnicholaus 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @jenadiusnicholaus, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**